esx_service/utils/auth_data.py

```diff
@@ -561,11 +568,43 @@ def handle_upgrade_1_1_to_1_2(self):
                      """
             sql_script = script.format(DB_MAJOR_VER, DB_MINOR_VER)
             self.conn.executescript(sql_script)
+
+            logging.debug("handle_upgrade_1_1_to_1_2: update vms table Done")
```

Comment:
I think upgrade logging can mainly go to info. Upgrade errors are notorious and we'd better have all data no matter what the logging level was. Plus, upgrade is one time event so we do not pollute the logs...